src/@providers/gcp/modules/vpc.ts

```diff
@@ -48,4 +48,6 @@ export class GoogleCloudVpcModule extends ResourceModule<'vpc', GoogleCloudCrede
       [this.getResourceRef(this.vpc)]: 'VPC',
     };
   }
+
+  hooks = {};
```

Comment:
This can still be removed :)

Reply:
nice find - stray change that got included in my function stuff 😛 Fixed this https://github.com/architect-team/arcctl/pull/83/commits/d5c8d8828419b848cffe37a9fa69388442183cf4